scripts/test.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
I don't think this is an official test, correct? More like a debug?

let's move to the debugging folder if so.

Reply:
This test was to see how `snakemake` injects the parameters into scripts. There was no debugging in here, just checking datatypes :) 

Originally, I will be doing this in a notebook to understand Snakemake's datatypes but it doesn't seem that `snakemake` does that. 

remove on commit:  d74575671a20747cfe0ac21744b162584cc915de